version: 1.0.4.0

main_timer_rate: 100

rc:
  channel_number: 6 # the same as RC mode (RC mode has to be switched off) 

service_call_attempts: 10

flight_constraints:
  type: fast
  use: true
